package.json

```diff
@@ -230,7 +230,7 @@
 		"express": "^4.17.1",
 		"express-rate-limit": "^5.2.6",
 		"fflate": "^0.7.1",
-		"fibers": "4.0.3",
+		"fibers": "^5.0.0",
```

Comment:
ideally we should use same version as Meteor with support to apple m1 `https://github.com/meteor/node-fibers/archive/refs/tags/5.0.0-1.tar.gz` but it didn't work when I tried